config/400-default-channel-config.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-channel-webhook
+  namespace: knative-eventing
+data:
+  # Configuration for defaulting channels that do not specify provisioners.
+  default-provisioner-name: in-memory-channel
```

Comment:
how about like this: https://github.com/knative/eventing/blob/master/config/400-flow-controller-config.yaml#L21

Reply:
@matzew I used that as inspiration for the current design, but I think @n3wscott has convinced me to allow per-namespace defaults, so it will look fairly different.

Reply:
Done. 

A single entry stores both the cluster default, as well as defaults for any namespace specified.